projects/cm_mcu/commands/I2CCommands.c

```diff
@@ -162,6 +162,7 @@ BaseType_t i2c_scan(int argc, char **argv, char *m)
   }
   if (acquireI2CSemaphore(s) == pdFAIL) {
     snprintf(m, SCRATCH_SIZE, "%s: could not get semaphore in time\r\n", argv[0]);
+    vTaskDelay(pdMS_TO_TICKS(330)); // delay 300 ms
```

Comment:
why do you delay here? and the comment is wrong (330 ms, not 300)

Reply:
you're right. we don't need this delay. sorry for an error, I will remove it.